patches/chrome-common-mac-app_mode_chrome_locator.mm.patch

```diff
@@ -1,13 +1,39 @@
 diff --git a/chrome/common/mac/app_mode_chrome_locator.mm b/chrome/common/mac/app_mode_chrome_locator.mm
-index efd272e706ba56b748e4b76b96d76cbd03aa07ac..b5d8d8cfa223d276db7a328036deeef658bc0320 100644
+index 7e5f3fa0b7e13065d4c28b9152522ee69f790d25..b974b7a6d1753527c53ea466fa3d80f4fc336ddf 100644
 --- a/chrome/common/mac/app_mode_chrome_locator.mm
 +++ b/chrome/common/mac/app_mode_chrome_locator.mm
-@@ -79,7 +79,7 @@ bool GetChromeBundleInfo(const base::FilePath& chrome_bundle,
-   // However, we want the shims to be agnostic to distribution and operate based
-   // on the data in their plist, so encode the framework names here.
-   NSDictionary* framework_for_exe = @{
--    @"Chromium": @"Chromium",
+@@ -113,7 +113,34 @@ bool GetChromeBundleInfo(const base::FilePath& chrome_bundle,
+     }
+   }
+ 
++<<<<<<< ours
+   if (!framework_path_and_structure)
```

Comment:
Ongoing merge work here?

Reply:
oh wow- this is interesting... Have never seen this before

Reply:
Thanks for proactively grabbing the review on this @yrliou 😄 Basically, the Jenkins job that @mkarolin and I had put together freaked out on the merge here 😂 Looks like this patch is no longer needed! Just removed in a commit and will let CI run